enrollment-server-onboarding/src/main/resources/application.properties

```diff
@@ -134,6 +134,8 @@ enrollment-server-onboarding.document-verification.zenid.restClientConfig.proxyH
 enrollment-server-onboarding.document-verification.zenid.restClientConfig.proxyPort=0
 enrollment-server-onboarding.document-verification.zenid.restClientConfig.proxyUsername=
 enrollment-server-onboarding.document-verification.zenid.restClientConfig.proxyPassword=
+enrollment-server-onboarding.document-verification.zenid.restClientConfig.connectionTimeout=10000
+enrollment-server-onboarding.document-verification.zenid.restClientConfig.responseTimeout=60000
```

Comment:
🤔 Just thinking about database transaction timeout, which is 30 seconds by default.

Reply:
Where do you get the information about 30 seconds? I believe the transaction timeout is longer than this, based on what we observed when testing the timeouts.

Reply:
Sorry, I have messed it up with JTA. It seems that the default timeout for user transactions in Spring is -1.